### PR TITLE
Give a hint about a Logger configuration.

### PR DESCRIPTION
**Pull request by franzholz**
It seems that not everything is done automatically.

### CODE REVIEW

File: Documentation/ApiOverview/Logging/Logger/Index.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ Use the :code:`LoggerAwareTrait` in your class to automatically instantiate :cod
    }
 
 
-Or, you can instantiate the Logger with :code:`makeInstance`.
+Or, you can instantiate the Logger with :code:`makeInstance`. Be aware that a simple Logger configuration must be made also.
```
Comment (mbrodala) [inline Documentation/ApiOverview/Logging/Logger/Index.rst:34]: That's actually not the case. You can retrieve a logger for anything without further configuration. By default the messages will end up in the default TYPO3 log this way. See `LogManager::getConfigurationForLogger()` which will traverse and at least use `$GLOBALS['TYPO3_CONF_VARS]['LOG']['writerConfiguration']`.

**Review by mbrodala — CHANGES_REQUESTED** [review]

**Review by franzholz — COMMENTED** [review]
So it is not required.

**Comment by mbrodala** [comment]
Exactly. If you have a specific issue, you can ask in the TYPO3 Slack.

**Comment by franzholz** [comment]
Should this not be documented also? Nobody knows if the configuration is necessary or not.

**Comment by mbrodala** [comment]
It is actually already mentioned, very first sentence:

> Instantiation of Loggers is configuration-free, as the LogManager automatically applies its configuration.

https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/Logging/Configuration/Index.html

**Pull request closed without merging**